static/app/components/events/autofix/autofixSolution.tsx

```diff
@@ -257,10 +410,56 @@ function AutofixSolutionDisplay({
   changesDisabled,
 }: Omit<AutofixSolutionProps, 'repos'>) {
   const {mutate: handleContinue, isPending} = useSelectSolution({groupId, runId});
-  const [isEditing, setIsEditing] = useState(false);
-  const [userCustomSolution, setUserCustomSolution] = useState('');
+  const [isEditing, _setIsEditing] = useState(false);
+  const [instructions, setInstructions] = useState('');
+  const [solutionItems, setSolutionItems] = useState<AutofixSolutionTimelineEvent[]>( // This will become outdated if multiple people use it, but we can ignore this for now.
+    () => {
+      // Initialize is_active to true for all items that don't have it set for backwards compatibility
+      return solution.map(item => ({
+        ...item,
+        is_active: item.is_active === undefined ? true : item.is_active,
+      }));
+    }
+  );
   const containerRef = useRef<HTMLDivElement>(null);
 
+  const handleAddInstruction = () => {
+    if (instructions.trim()) {
+      // Create a new step from the instructions input
+      const newStep: AutofixSolutionTimelineEvent = {
+        title: instructions,
+        timeline_item_type: 'human_instruction',
+        is_most_important_event: false,
+        is_active: true,
+      };
+
+      // Add the new step to the solution
+      setSolutionItems([...solutionItems, newStep]);
+
+      // Clear the input
+      setInstructions('');
+    }
+  };
+
+  const handleKeyDown = (e: React.KeyboardEvent<HTMLInputElement>) => {
+    if (e.key === 'Enter') {
+      e.preventDefault();
+      handleAddInstruction();
+    }
+  };
+
```

Comment:
would it be cleaner to just use an html form?

Reply:
yeah